book/src/02_development/01_tools.md

```diff
@@ -27,7 +27,8 @@ Create a provider test execution report for the candidate branch:
 yarn build && yarn test:provider --reporter json | tee candidate-test-provider-logs.json
 ```
 
-Generate a comparison report that will list slower tests in the candidate branch:
+Generate a comparison report that will list slower tests in the candidate
```

Comment:
It seems like some form of formatter was triggered. To simplify merge requests, each sentence was on a separate line. That's no longer the case.

Reply:
Thanks fixed in [daa0615](https://github.com/NomicFoundation/hardhat/pull/4978/commits/daa061553a1a33cecfe7e55f4fe978fc1e8980fc)